app/controllers/groups_controller.rb

```diff
@@ -38,7 +41,12 @@ def member_listing
     @group = Group.friendly.find(params[:id])
   end
 
+  def settings
+    @group = Group.all
+  end
+
   def group_params
     params.require(:group).permit(:name, :description, :neighborhood_id)
```

Comment:
@nikitr You need to add :kind to the params. This currently isn't working. Tested through creation